corehq/ex-submodules/casexml/apps/case/xform.py

```diff
@@ -23,6 +26,41 @@
 from dimagi.utils.logging import notify_exception
 
 
+class DirtinessFlag(object):
```

Comment:
:) wasn't being pushy, just saw something that I thought was interesting that reminded me of it and wanted to write down somewhere